package.json

```diff
@@ -10,26 +10,34 @@
   "bugs": {
     "url": "https://github.com/wmonk/create-react-app/issues"
   },
-  "files": ["bin", "config", "scripts", "template", "utils"],
+  "files": [
+    "bin",
+    "config",
+    "scripts",
+    "template",
+    "utils"
+  ],
   "bin": {
     "react-scripts-ts": "./bin/react-scripts-ts.js"
   },
   "dependencies": {
     "autoprefixer": "7.1.6",
-    "babel-jest": "20.0.3",
+    "babel-core": "6.26.3",
+    "babel-jest": "23.4.2",
     "babel-loader": "7.1.2",
     "babel-preset-react-app": "^3.1.2",
+    "babel-runtime": "6.26.0",
     "case-sensitive-paths-webpack-plugin": "2.1.1",
     "chalk": "1.1.3",
     "css-loader": "0.28.7",
     "dotenv": "4.0.0",
     "dotenv-expand": "4.2.0",
     "extract-text-webpack-plugin": "3.0.2",
     "file-loader": "1.1.5",
-    "fork-ts-checker-webpack-plugin": "^0.2.8",
+    "fork-ts-checker-webpack-plugin": "0.4.4",
```

Comment:
The latest version is `0.4.9`.

Reply:
On master this dependency version was upgraded and then downgraded back to 0.2.x, any idea why?